TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -765,8 +841,7 @@ private static SymbolReference ResolveProcedureName(SymbolTable table, SymbolRef
                 }
                 else
                 {
-                    // Use or Exec node : statement
-                    Debug.Assert(child is Use || child is Exec);
```

Comment:
Code has been removed here but it shouldn't.
This is related to PR #1553
Wait for PR #1553 to be merged into develop then.

Reply:
ok